README.md

```diff
@@ -13,9 +13,11 @@
 [![slack-community](https://img.shields.io/badge/Slack-4A154B?style=plastic&logo=slack&logoColor=white)](https://slack.bridgecrew.io/)
  
 
-**Checkov** is a static code analysis tool for infrastructure-as-code.
+**Checkov** is a static code analysis tool for infrastructure-as-code and it is also a software-composition-analysis tool for open-source images and packages.
```

Comment:
Using dashes is pretty uncommon and we scan closed source images. Suggest:
`**Checkov** is a static code analysis tool for infrastructure as code (IaC) and also a software composition analysis (SCA) tool for images and open source packages.`